ports/seal/portfile.cmake

```
@@ -1,43 +1,49 @@
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+if(VCPKG_TARGET_IS_WINDOWS) # Win32 and dynamic
+    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+endif()
```

```suggestion
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```
This change seems to be unnecessary. Since vcpkg only supports static build on Unix platform. Sorry, I didn't notice this point before.